src/unstract/sdk/adapters/llm/palm/src/palm.py

```diff
@@ -1,12 +1,12 @@
 import os
-from typing import Any
+from typing import Any, Optional
 
+from google.api_core.exceptions import GoogleAPICallError
```

Comment:
[MINOR] Are we adding the dependency to the toml, since we refer from library?

Reply:
@harini-venkataraman it comes from our llama index dependencies
![image](https://github.com/user-attachments/assets/0b47a84b-138c-4439-b74f-bef0088c85fc)